### PR TITLE
[Merged by Bors] - chore: Rename `Set.Sigma` to `Set.sigma`

### PR DESCRIPTION
**Pull request by YaelDillies**
This was a misport.

Also adds missing whitespace after some `Σ`s while we're here, and some other arbitrary line re-wrappings.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by alreadydone — COMMENTED** [review]
Why all the changes from theorem to lemma?

**Comment by YaelDillies** [comment]
Oh because why not. I can drop them if you want.

**Comment by eric-wieser** [comment]
Yes, please drop them. We're eventually going to do a global sweep (pending some meta code first), which means there's no point cluttering diffs in the meantime.

**Comment by alreadydone** [comment]
[Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/lemma.20vs.20theorem/near/391732529) for reference

File: Mathlib/Data/Set/Sigma.lean [hunk 1]
```diff
@@ -51,62 +51,52 @@ theorem image_sigmaMk_preimage_sigmaMap {β : ι' → Type*} {f : ι → ι'} (h
...
 theorem exists_sigma_iff {p : (Σi, α i) → Prop} :
-    (∃ x ∈ s.Sigma t, p x) ↔ ∃ i ∈ s, ∃ a ∈ t i, p ⟨i, a⟩ :=
+    (∃ x ∈ s.sigma t, p x) ↔ ∃ i ∈ s, ∃ a ∈ t i, p ⟨i, a⟩ :=
   ⟨fun ⟨⟨i, a⟩, ha, h⟩ ↦ ⟨i, ha.1, a, ha.2, h⟩, fun ⟨i, hi, a, ha, h⟩ ↦ ⟨⟨i, a⟩, ⟨hi, ha⟩, h⟩⟩
 #align set.exists_sigma_iff Set.exists_sigma_iff
 
-@[simp]
-theorem sigma_empty : (s.Sigma fun i ↦ (∅ : Set (α i))) = ∅ :=
-  ext fun _ ↦ and_false_iff _
+@[simp] theorem sigma_empty : s.sigma (fun i ↦ (∅ : Set (α i))) = ∅ := ext fun _ ↦ and_false_iff _
```
Comment (eric-wieser) [inline Mathlib/Data/Set/Sigma.lean:84]: Rewrapping lines here and elsewhere seems kind of unnecessary for a PR that just replaced `S` with `s` (or even just in general): but it's probably not worth reverting, so I've just added to the PR description

bors  d+

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**